go-controller/pkg/ovn/ovn.go

```diff
@@ -104,7 +119,13 @@ func (oc *Controller) WatchEndpoints() {
 				logrus.Errorf("Error in adding load balancer: %v", err)
 			}
 		},
-		UpdateFunc: func(old, new interface{}) { return },
+		UpdateFunc: func(old, new interface{}) {
+			ep := new.(*kapi.Endpoints)
+			err := oc.addEndpoints(ep)
```

Comment:
May be we should check the ip subsets between old and new objects, so that we do not have to employ a DeepEqual on the cached data downstream. 
Something to consider, though this PR looks good enough.